src/DirectionProvider.jsx

```diff
@@ -4,15 +4,15 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import { forbidExtraProps } from 'airbnb-prop-types';
+import { explicitNull, forbidExtraProps, or } from 'airbnb-prop-types';
 import brcast from 'brcast';
 import brcastShape from './proptypes/brcast';
 import directionPropType from './proptypes/direction';
-import { DIRECTIONS, CHANNEL } from './constants';
+import { DIRECTIONS, CHANNEL, defaultDirection } from './constants';
 
 const propTypes = forbidExtraProps({
   children: PropTypes.node.isRequired,
-  direction: directionPropType.isRequired,
+  direction: or([directionPropType, explicitNull()]).isRequired,
```

Comment:
I suspect we may need `or([directionPropType.isRequired, explicitNull()])` here?